includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -2299,107 +2299,99 @@ public function process_checkout_session_redirect( int $order_id, bool $is_pay_f
+			&& ! empty( $error_code )
+			&& ! in_array( $error_code, $cancellation_codes, true );
+
+		if ( $is_hard_failure ) {
+			WC_Stripe_Logger::error(
+				'Checkout session redirect failed for order: ' . $order_id,
+				[
+					'session_status' => $status,
+					'error_code'     => $error_code,
+					'error_message'  => $error_message,
+				]
+			);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'process_checkout_session_redirect'. (undefined)

(StaticAccess)